### ISSUE TITLE
Properties Resources and Settings are not nested

### ISSUE THREAD

**Issue by njannink** [aienabled/WpfNetStandardSample#4]
This looks really promising and I'm trying to use it. One thing I found is that resources and settings are not nested by default. I had to add the following lines to my csprojs  but maybe you can include this in the base templates somehow.

```
    <!-- Resources -->
    <EmbeddedResource Update="Properties\Resources.resx" Generator="ResXFileCodeGenerator" LastGenOutput="Resources.Designer.cs" />
    <Compile Update="Properties\Resources.Designer.cs" AutoGen="True" DependentUpon="Resources.resx" DesignTime="True" />

    <!-- Settings -->
    <None Update="Properties\Settings.settings" Generator="SettingsSingleFileGenerator" LastGenOutput="Settings.Designer.cs" />
    <Compile Update="Properties\Settings.Designer.cs" AutoGen="True" DependentUpon="Settings.settings" />
```

**Comment by aienabled** [comment]
@njannink thanks for reporting!
I've tried to add support for these item types but unfortunately VS continue ignoring the ContentType definitions from my ProjectItemsSchema.xaml (for some reason it works fine for XAML "Page" item type but not for EmbeddedResource).

I've noticed that if you're adding them via VS "Add item" they're added properly to csproj file with all the required properties. And *.Designer.cs files are properly displayed as the nested items. So it's not a big deal.

Regards!
